### PR TITLE
Volunteers

### PR DESCRIPTION
**Pull request by mikestephens**

### CODE REVIEW

File: API_DOC.md [hunk 1]
```diff
@@ -193,3 +198,76 @@ params :verified      Boolean, default: false, desc: 'Whether the posting is ver
...
+```
+api :GET, '/volunteer'
+param :page     Integer, desc: "What page to get"
+param :per_page Integer, default: 25 desc: "How many results per page"
+```
+### Filters
+EX: `filter[FILTER_NAME]=FILTER_VALUE`
+
+Filterable Values:
+* `:city, type: :string`
```
Comment (mboperator) [inline API_DOC.md]: Seems like this is the housing#index endpoint, not volunteers

File: API_DOC.md [hunk 2]
```diff
@@ -193,3 +198,76 @@ params :verified      Boolean, default: false, desc: 'Whether the posting is ver
...
+## Show
+```
+api :GET, '/volunteer/:id'
+```
+## Create
+```
+api :POST, '/volunteer'
+param :skills                 Text,  required: true
+param :number_of_volunteers   Integer, required: 'organization'
+param :volunteer_notes        Text, required: 'organization'
```
Comment (mboperator) [inline API_DOC.md]: Can i ask a clarifying question? Does everything labeled with required: organization require an organization to be present? Do those represent an organization based volunteer opportunity?

File: API_DOC.md (hunk 1 above)
Comment (mikestephens) [reply API_DOC.md]: Hehe you are right. The docs are wrong but the filters are right

Comment (mikestephens) [reply API_DOC.md]: Filters meaning in the actual code

**Pull request merged**